pinot-plugins/pinot-timeseries-lang/pinot-timeseries-m3ql/src/main/java/org/apache/pinot/tsdb/m3ql/plan/TransformNullPlanNode.java

```diff
@@ -34,15 +34,20 @@ public class TransformNullPlanNode extends BaseTimeSeriesPlanNode {
 
   @JsonCreator
   public TransformNullPlanNode(@JsonProperty("id") String id, @JsonProperty("defaultValue") Double defaultValue,
-      @JsonProperty("children") List<BaseTimeSeriesPlanNode> children) {
-    super(id, children);
+      @JsonProperty("inputs") List<BaseTimeSeriesPlanNode> inputs) {
```

Comment:
Why are we changing children to inputs? Curious, does it have any significance ?

Reply:
It is to be consistent with pinot-query-planner PlanNode, which is what we want to coalesce with in the end.